examples/error-reporting/error_reporting.md

```diff
@@ -1,10 +1,19 @@
 # Errors
+In Ballerina invalid states are represented by `error` values. Each error value has,
+1. Message, a human-readable `string` describing the error.
+2. Cause, which is an `error` value if this error was caused by another error. Otherwise `nil`.
```

Comment:
```suggestion
2. Cause, which is an `error` value if this error was caused by another error, otherwise nil.
```

We only emphasize values, names, keywords, etc.